examples/src/bin/simpleserver.rs

```diff
@@ -0,0 +1,53 @@
+//! This is the simplest possible server using rustls that does something useful:
+//! it accepts the default configuration, loads a server certificate and private key,
+//! and then accepts a single client connection.
+//!
+//! You must either set the CERTFILE and PRIV_KEY_FILE env vars to point to a server
+//! certificate and private key, or place 'localhost.pem' and 'localhost-key.pem' in
+//! the directory you run this example from.
+//!
+//! Note that `unwrap()` is used to deal with networking errors; this is not something
+//! that is sensible outside of example code.
+
+use std::error::Error as StdError;
+use std::fs::File;
+use std::io::{BufReader, Read, Write};
+use std::net::TcpListener;
+use std::sync::Arc;
+
+use rustls::ServerConnection;
```

Comment:
I would drop this import and fully qualify the single use on line 42, for consistency with `rustls::ServerConfig` on line 35. Generally I like to write out full names in example code, where that doesn't cause immensely long lines or otherwise hide the purpose of the example.

Reply:
Good point, fixed.